### DISCUSSION TITLE
Having all examples from Tutorials available

### DISCUSSION THREAD

**Discussion by aboryseenko** [thliebig/openEMS-Project#51] [General]
Could you please make possible more examples from Wiki Tutorials available? 
Specifically, I am interested in Tutorial: Coax to Waveguide Adapter which linked to unworking host website. In this example, I do not understand how to control mesh for the coax portion since it is axially symmetrical and requires much higher mesh resolution unlike the rectangular waveguide.  It is especially demanding for small diameter coaxes like those, for example, from N-type connector on WR340 waveguides and similar. 
Thank you!

**Comment by biergaizi** [comment]
@thliebig Do you still have the file for the "Coax to Waveguide Adapter" example in your hard drive somewhere? This file was previously hosted on the forum, and now it has permanently gone after the server crash.

**Reply by 0xCoto** [reply]
https://wiki.openems.de/index.php/Tutorial:_Coax_to_Waveguide_Adapter.html

**Reply by thliebig** [reply]
No I don't think that I still have it... the web archive was no help either...
